packages/fxa-content-server/app/scripts/lib/router.js

```diff
@@ -179,36 +173,32 @@ Router = Router.extend({
     },
     'connect_another_device(/)': createViewHandler(ConnectAnotherDeviceView),
     'cookies_disabled(/)': function () {
-      this.createReactOrBackboneViewHandler(
-        'cookies_disabled',
-        CookiesDisabledView,
-        {
-          // HACK: this page uses the history API to navigate back and must go back one page
-          // further if being redirected from content-server. Flow params are not always
-          // available to check against, so we explicitly send in an additional param.
-          contentRedirect: true,
-          // We pass along `disable_local_storage` for functional-tests that hit another page
-          // with this param (synthetically disabling local storage). Without this, tests will
-          // be redirected to this page, but local storage will appear enabled.
-          /* eslint-disable camelcase */
-          ...(Url.searchParam(
-            'disable_local_storage',
-            this.window.location.search
-          ) === '1' && {
-            disable_local_storage: 1,
-          }),
-        }
-      );
+      this.createReactViewHandler('cookies_disabled', {
+        // HACK: this page uses the history API to navigate back and must go back one page
+        // further if being redirected from content-server. Flow params are not always
+        // available to check against, so we explicitly send in an additional param.
+        contentRedirect: true,
+        // We pass along `disable_local_storage` for functional-tests that hit another page
+        // with this param (synthetically disabling local storage). Without this, tests will
+        // be redirected to this page, but local storage will appear enabled.
+        /* eslint-disable camelcase */
+        ...(Url.searchParam(
+          'disable_local_storage',
+          this.window.location.search
+        ) === '1' && {
+          disable_local_storage: 1,
+        }),
+      });
     },
     'force_auth(/)': createViewHandler(ForceAuthView),
     'inline_totp_setup(/)': createViewHandler(InlineTotpSetupView),
     'inline_recovery_setup(/)': createViewHandler(InlineRecoverySetupView),
     'legal(/)': function () {
-      this.createReactOrBackboneViewHandler('legal', 'legal');
+      this.createReactViewHandler('legal');
     },
     'legal/privacy(/)': createViewHandler('pp'),
```

Comment:
Ack, I don't know why this is showing `createViewHandler` for `legal/privacy`, I feel like even QA verified the React version was showing here? 🥴 I think you need to change this one too to `this.createReactViewHandler`. We may want QA to look at that page with the next tag too in case this page wasn't tested somehow before...

Reply:
😅 ack!